### ISSUE TITLE
Expired asciinema in readme

### ISSUE THREAD

**Issue by pitasi** [Minigugus/bazx#2]
At first I thought that this repo was archived/abandoned or something, then I realized that it's just the asciinema in the readme that expired :)

It shows up like that:
![image](https://user-images.githubusercontent.com/15048443/145820412-88ad40b8-98a8-491c-9477-05d63c52fc31.png)

the asciinema link says that recordings older than 7 days are archived: https://asciinema.org/a/ydfYbBXFyDDyDOeSPormkjEo6.

**Comment by Minigugus** [comment]
Indeed, recordings only last for 7 days without an account. When I chose asciinema, I thought Github would render the recording interactively but no, it's just an image. Then, maybe a GIF or just a nice screen could be better? I'm not sure what would be best, do you have any preference?

**Comment by pitasi** [comment]
I'm not sure what was being showed in the video, if it can fit a screenshot I think it would be better because it shows everything at a glance; if not, a good old gif will be fine 😃
